src/components/process-accordion/process-accordion.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+} from '@reach/accordion';
+import '@reach/accordion/styles.css';
+import styled from 'styled-components';
+import { CaptionText } from '../typography/typography';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+import { LaunchingRocket } from '../icons/launching-rocket';
+import { Circle } from '../icons/circle';
+import { FlyingRocket } from '../icons/flying-rocket';
+import { FlagOnEarth } from '../icons/flag-on-earth';
+import { Satellite } from '../icons/satellite';
+
+const ProcessCaption = styled(CaptionText)`
+  font-size: 16px;
+  margin: 16px;
+`;
+
+const StyledAccordion = styled(Accordion)`
+  display: relative;
+`;
+
+const StyledAccordionPanel = styled(AccordionPanel)`
+  padding: 16px;
+  padding-top: 8px;
+  font-size: 16px;
+  line-height: 150%;
+  white-space: pre-line;
+
+  > svg {
+    display: block;
+    margin-bottom: 16px;
+  }
+`;
+
+const StyledAccordionButton = styled(AccordionButton)`
+  background: none;
+  border: none;
+  padding: 0px;
+
+  display: relative;
+  width: 100%;
+  cursor: pointer;
+  border-top: 1px solid rgba(255, 255, 255, 0.1);
+  text-align: left;
+`;
+
+const StyledAccordionItem = styled(AccordionItem)`
+  &:hover {
+    background: linear-gradient(
+      180deg,
+      rgba(77, 121, 255, 0.1) 0%,
+      rgba(77, 121, 255, 0) 100%
+    );
+  }
+
+  &:last-child {
+    border-bottom: 1px solid rgba(255, 255, 255, 0.1);
+  }
+
+  &[data-state='open'] {
+    position: relative;
+    &::after {
+      content: url("data:image/svg+xml;utf8,<svg width='12' height='8' viewBox='0 0 12 8' fill='none' xmlns='http://www.w3.org/2000/svg'>
```

Comment:
I don't like those inline icons. Let's refactor so we can use icon components instead.